dts/bindings/mfd/nxp,sc18is606.yaml

```
@@ -0,0 +1,33 @@
+description: |
+  NXP SC18IS606 I2C to SPI/GPIO bridge.
+
+  The SC18IS606 is supports both an external SPI and GPIO controller.
+  These controllers have to be added  to the Device Tree as children, While
+  the device itself has to be a child of a I2C controller.
+
+  An example configuration:
+
+  &i2c0 {
+    status = "okay";
+    pinctrl-0 = <&i2c0_default>;
+    pinctrl-names = "default";
+
+    sc18is606: sc18is606 {
+      compatible ="nxp,sc18is606";
```

nit, but you have the same minor formatting issue in other places 

```suggestion
      compatible = "nxp,sc18is606";
```